pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1155,6 +1155,37 @@ func initializeQEMUCmdAndQEMUArg(domain *api.Domain) {
 	}
 }
 
+func isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool {
+	//usb controller is turned on, only when user specify input device with usb bus,
+	//otherwise it is turned off
+	//In ppc64le usb devices like mouse / keyboard are set by default,
+	//so we can't disable the controller otherwise we run into the following error:
+	//"unsupported configuration: USB is disabled for this domain, but USB devices are present in the domain XML"
+	if !isAMD64(c.Architecture) {
```

Comment:
@zhlhahaha I wonder if we thought about this place also for arm.

Reply:
Yes, that'd be interesting, I've left it as it was but this check could be specialized for ppc64le if there's no need for it in other arches different than x86_64.

Reply:
Usb storage is not supported by the qemu-kvm binary on Arm64, we only support following storage devices.
```
Storage devices:
name "scsi-block", bus SCSI, desc "SCSI block device passthrough"
name "scsi-cd", bus SCSI, desc "virtual SCSI CD-ROM"
name "scsi-disk", bus SCSI, desc "virtual SCSI disk or CD-ROM (legacy)"
name "scsi-generic", bus SCSI, desc "pass through generic scsi device (/dev/sg*)"
name "scsi-hd", bus SCSI, desc "virtual SCSI disk"
name "vhost-user-fs-device", bus virtio-bus
name "vhost-user-fs-pci", bus PCI
name "virtio-blk-device", bus virtio-bus
name "virtio-blk-pci", bus PCI, alias "virtio-blk"
name "virtio-blk-pci-non-transitional", bus PCI
name "virtio-blk-pci-transitional", bus PCI
name "virtio-scsi-device", bus virtio-bus
name "virtio-scsi-pci", bus PCI, alias "virtio-scsi"
name "virtio-scsi-pci-non-transitional", bus PCI
name "virtio-scsi-pci-transitional", bus PCI
```

Reply:
There are so many limitation caused by qemu-kvm binary on Arm64, so I think if it is possible to build qemu-kvm in a normal way rather than removing so many devices. @rmohr

Reply:
@zhlhahaha do we need to fail admission of VMIs with USB disks on Arm64 then?

Reply:
> @zhlhahaha do we need to fail admission of VMIs with USB disks on Arm64 then?

I think so.

Reply:
Ah! I think this PR will do: https://github.com/kubevirt/kubevirt/pull/7007

Reply:
> Ah! I think this PR will do: #7007

Yes